content/blog/2021-11-30-november-21-community-gems.md

```diff
@@ -0,0 +1,192 @@
+---
+title: November '21 Community Gems
+date: 2021-11-30
+description: |
+  A roundup of technical Q&A's from the DVC and CML community. This month: CML runners, working with data, DVC Studio, and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC and CML community. This month: CML runners, working with data, DVC Studio, and more.
+picture: 2021-11-30/nov-community-gems.png
```

Comment:
image is missing still

Reply:
Yes, that is still left as a thing to update in the PR description. @jendefig has requested it and it should be ready by the time we merge tomorrow.